### DISCUSSION TITLE
Version with id [x] doesn't exist

### DISCUSSION THREAD

**Discussion by ghost** [pimcore/pimcore#17549] [Q&A / Support]
When I create objects via console application, I get the following message in the "Versions Tab": **"Version with id [x] doesn't exist"**

**Comment by hoangngx** [comment]
I got the same problem when trying to update object via console

**Comment by rndDev615** [comment]
Same problem here :/

**Comment by rndDev615** [comment]
you have to adjust the permissions for /var/versions/objects
